components/SubmissionComments.tsx

```diff
@@ -53,16 +128,54 @@ export const SubmissionComments: React.FC<{
               inline
             />
             {id === c.authorId ? (
-              <div>
-                <Button
-                  outline
-                  type="danger"
-                  color="danger"
-                  onClick={() => handleDeleteComment(c)}
-                >
-                  <TrashIcon />
-                </Button>
-              </div>
+              editing === i ? (
+                <div>
+                  <Button
+                    type="light"
+                    color="black"
+                    onClick={() => {
+                      handleDiscardChange()
+                    }}
+                    data-testid="discard-button"
+                  >
+                    Discard Changes
```

Comment:
I think this button needs to emphasize that what the user is about to do will remove what they've written in the input box. What do you think?

It could do so by making the button style `danger` like the `UNLINK DISCORD` button.

<img width="315" alt="image" src="https://user-images.githubusercontent.com/35906419/189523384-88d0c286-4d1a-4fd5-9590-2f75d51c2b7a.png">

<img width="189" alt="image" src="https://user-images.githubusercontent.com/35906419/189523489-451b584a-ba6c-4d83-a5de-d28f6f4212ca.png">